docs/source/use_with_pytorch.mdx

```diff
@@ -0,0 +1,194 @@
+# Use with PyTorch
+
+## Tensors
```

Comment:
I think we can remove the `tensors` header since it doesn't have any associated content under it. It would also be nice to write a brief intro under `# Use with PyTorch` about what this doc covers and how it will help users.